synapse/events/spamcheck.py

```diff
@@ -17,24 +17,25 @@
 import inspect
 from typing import Any, Dict, List, Optional, Tuple
 
-from synapse.spam_checker_api import RegistrationBehaviour, SpamCheckerApi
+from synapse.spam_checker_api import RegistrationBehaviour
 from synapse.types import Collection
 
 MYPY = False
 if MYPY:
+    import synapse.events
```

Comment:
Why this change?

Reply:
well, because `synapse.events` is referenced at line 42. I'm not really sure why mypy wasn't complaining before, but I think including it here is more correct.

Reply:
Looks like mypy checks this file. 🤷